torch/distributed/autograd/__init__.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+class context(object):
+    '''
+        Autograd context object to wrap forward and backward passes when using
```

Comment:
Comments in other py files usually don't have an additional indention here [[example](https://github.com/pytorch/pytorch/blob/master/torch/distributed/distributed_c10d.py#L322)].